src/adaptation/stepsize.jl

```diff
@@ -88,7 +88,7 @@ NesterovDualAveraging(
     t_0::T,
     κ::T,
     δ::T,
-    ϵ::VT,
+    ϵ::VT
```

Comment:
Unrelated change it seems:
```suggestion
    ϵ::VT,
```